### PR TITLE
AC Test Fix - LRAC-9388 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-9388 (fix)
[afonsoshsantos#21](https://github.com/AfonsoHSantos/liferay-portal/pull/21)
https://testray.liferay.com/home/-/testray/case_results?tab=runs&testrayBuildId=1564830182 (run)

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase [hunk 1]
```diff
@@ -69,24 +61,12 @@ definition {
...
-		}
-
-		task ("Switch the property in AC") {
-			ACUtils.launchAC();
-
-			ACProperties.switchProperty(propertyName = "${propertyName}");
-		}
-
 		task ("Create Access Request") {
 			ACUtils.launchAC();
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase:65]: Hi @AfonsoHSantos, no need to launch the AC again

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase [hunk 2]
```diff
@@ -97,32 +77,20 @@ definition {
...
-		}
-
-		task ("Switch the property in AC") {
-			ACUtils.launchAC();
-
-			ACProperties.switchProperty(propertyName = "${propertyName}");
-		}
-
 		task ("Create Delete Request") {
 			ACUtils.launchAC();
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase:89]: Hi @AfonsoHSantos, no need to launch the AC again

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase [hunk 3]
```diff
@@ -133,29 +101,20 @@ definition {
...
 	@description = "Automation ID: LRAC-8109 | Test Summary: Create Suppress Request"
 	@priority = "5"
 	test CanCreateSuppressRequest {
-		task ("Get site assigned property name") {
-			var propertyName = ACDXPSettings.getAssignedPropertyName();
-		}
-
-		task ("Switch the property in AC") {
+		task ("Go to settings and create a suppress request") {
 			ACUtils.launchAC();
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase:113]: Hi @AfonsoHSantos, no need to launch the AC again

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase [hunk 4]
```diff
@@ -179,17 +138,9 @@ definition {
 		// AC Refactor: Timing issues may happen on this test case
 		// AC Refactor ticket: LRAC-9439
 
-		task ("Get site assigned property name") {
-			var propertyName = ACDXPSettings.getAssignedPropertyName();
-		}
-
-		task ("Switch the property in AC") {
+		task ("Go to settings and create a suppress request") {
 			ACUtils.launchAC();
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase:142]: Hi @AfonsoHSantos, no need to launch the AC again

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase [hunk 5]
```diff
@@ -241,16 +192,9 @@ definition {
 		// AC Refactor: Timing issues may happen on this test case
 		// AC Refactor ticket: LRAC-9438
 
-		task ("Get site assigned property name") {
-			var propertyName = ACDXPSettings.getAssignedPropertyName();
-		}
-
-		task ("Switch the property in AC") {
+		task ("Go to settings and create a suppress request") {
 			ACUtils.launchAC();
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataControlAndPrivacy.testcase:196]: Hi @AfonsoHSantos, no need to launch the AC again

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by christianstokes** [comment]
ci:forward

**Pull request closed without merging**